CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All Notable changes to `League\Container` will be documented in this file
 
+## 1.3.0 - 2015-01-30
```

Comment:
A good habit for putting changelog items in place is to use `Unreleased` as the header instead, as this could be a pain and conflicty.

No other opinions on this PR.